### ISSUE TITLE
No hint if there is no search result?

### ISSUE THREAD

**Issue by qiaoruntao** [zealdocs/zeal#558]
I'm using win10.
It seems zeal doesn't give any notice when there are no search result.
This could result in some misunderstanding.
Maybe there should be some enhancement :)

**Comment by Sathors** [comment]
@qiaoruntao , when you are typing, you can see the results filtering down. So if you have nothing in the left pane, it means nothing has been found.

What kind of notice would you be hoping for?

**Comment by qiaoruntao** [comment]
The result part is blank during the search as well as when there are no search result. And the search can take a long time if there are too many docsets.
For myself, I downloaded all the docsets in case I would need them. The search result would take a long time to show up and It would be hard to find out whether it is still searching or out. In this situation, there could be a misunderstanding between these two states.
So I think it would be better if there is a notice to tell you if there isn't any search results.

**Comment by trollixx** [comment]
Thanks for the suggestion. I'll add some sort of indication if there are no results.

**Comment by qiaoruntao** [comment]
Thanks! That would be great!

**Comment by Sathors** [comment]
Ok, I did not think about the case with too many docsets.

But are you sure you need to download ALL of them???
Because the day you need them, they are only a click away (with internet connection though).

@trollixx instead of an indication that there are no result, maybe you could think of an indication that zeal is currently searching.

**Comment by qiaoruntao** [comment]
Yes. I gonna have an offline exam so I'd like to download all docsets in case I need any of them.
Add an indication while searching or when search is end and there are no result makes not much difference. They are both some kind of way to identify those two conditions.
